common/src/user-notification-preferences.ts

```diff
@@ -7,44 +7,44 @@ import { PrivateUser } from './user'
 export type notification_destination_types = 'email' | 'browser' | 'mobile'
 export type notification_preference = keyof notification_preferences
 export type notification_preferences = {
-  // Watched Markets
-  all_comments_on_watched_markets: notification_destination_types[]
-  all_answers_on_watched_markets: notification_destination_types[]
+  // Watched Questions
+  all_comments_on_watched_questions: notification_destination_types[]
```

Comment:
Changing properties that are stored in the database would break things unless we also ran a script to change them. Probably not worth the effort.

Notification preferences fit this scenario, I think.

I wonder which other changes in the PR are like this?